config/config-feature-flags.yaml

```diff
@@ -142,3 +139,5 @@ data:
   enable-concise-resolver-syntax: "false"
   # Setthing this flag to "true" will enable native Kubernetes Sidecar support
   enable-kubernetes-sidecar: "false"
+  # Setting this flag to "true" will have no effect since StepActions are a stable feature
```

Comment:
NIT: I would say "Setting this flag to "false" will have no effect...

Reply:
updated